Src/WitsmlExplorer.Frontend/contexts/queryContext.tsx

```diff
@@ -87,7 +147,7 @@ export function QueryContextProvider({ initialQueryState, children }: QueryConte
   useEffect(() => {
     const dispatch = setTimeout(() => {
       setStoredQuery(queryState);
-    }, 200);
+    }, 1000);
```

Comment:
The purpose is to avoid writing constantly to localStorage for every letter added to the query. With the timeout it should just be updated 1 sec after the last change. If another change is added before the 1 sec timer runs out, it clears the previous timeout and creates a new one. 1 sec might be slightly much, so I'll change it to 500ms. I don't think the user will have time to leave the page after making a change before the 500ms timer runs out.